model/permission.go

```diff
@@ -641,6 +644,22 @@ func initializePermissions() {
 		PERMISSION_DEMOTE_TO_GUEST,
 		PERMISSION_USE_CHANNEL_MENTIONS,
 	}
+
+	CHANNEL_MODERATED_PERMISSIONS = []string{
+		PERMISSION_CREATE_POST.Id,
+		"create_reactions",
+		"manage_members",
+		PERMISSION_USE_CHANNEL_MENTIONS.Id,
+	}
+
+	CHANNEL_MODERATED_PERMISSIONS_MAP = map[string]string{
+		PERMISSION_CREATE_POST.Id:                    CHANNEL_MODERATED_PERMISSIONS[0],
+		PERMISSION_ADD_REACTION.Id:                   CHANNEL_MODERATED_PERMISSIONS[1],
+		PERMISSION_REMOVE_REACTION.Id:                CHANNEL_MODERATED_PERMISSIONS[1],
+		PERMISSION_MANAGE_PUBLIC_CHANNEL_MEMBERS.Id:  CHANNEL_MODERATED_PERMISSIONS[2],
+		PERMISSION_MANAGE_PRIVATE_CHANNEL_MEMBERS.Id: CHANNEL_MODERATED_PERMISSIONS[2],
+		PERMISSION_USE_CHANNEL_MENTIONS.Id:           CHANNEL_MODERATED_PERMISSIONS[3],
+	}
```

Comment:
This map is required to map existing permissions with their moderated permission name
eg. `remove_reaction` and `add_reaction` both get mapped to `create_reactions`